src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const normalRobot = {};
+
+  for (const i in robot) {
```

Comment:
```suggestion
  for (const key in robot) {
```
Було б правильніше цю змінну назвати key, адже ми перебираємо ключі. Якщо в нас `i` - то це дає зрозуміти, що в нас індекс. Тут же індексів немає